test/Behavior/BehaviorTest.php

```diff
@@ -122,11 +140,14 @@ public function test_StatusCodes_Success() {
 			->once();
 
 		// when
-		$output = $manager->process();
+		$manager->process();
```

Comment:
Glad to remove vars never used. Did we want to test/interrogate $output in a `/then`?

Reply:
We do in the other tests. I only removed it from ones not using it. Most that had it but don't need it are expecting exceptions.